client/web/web.go

```diff
@@ -272,18 +280,8 @@ req.send(null);
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if s.devMode {
 		if r.URL.Path == "/api/data" {
-			user, err := authorize(w, r)
-			if err != nil {
-				return
-			}
-			switch r.Method {
-			case httpm.GET:
-				s.serveGetNodeDataJSON(w, r, user)
-			case httpm.POST:
-				s.servePostNodeUpdate(w, r)
-			default:
-				http.Error(w, "method not allowed", http.StatusMethodNotAllowed)
-			}
+			// Pass through to other handlers via CSRF protection.
```

Comment:
same... this `/api/data` can probably be exposed outside of devmode.  Nothing in the current static web client is calling it, but there's really no harm in exposing it... it's the same data we put in the go template anyway.

Reply:
Talked live, leaving this behind `--dev` for now because we've got the `POST` endpoint hooked up already. Will move it out when we hook up posting from the browser side.